spec/player_turn_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "../lib/player_turn"
+
+RSpec.describe PlayerTurn do
+  describe "#get_coordinates" do
```

Comment:
Nice! There is some complex logic in this universe of taking a turn, and having these collaborators with their own responsibilities allows these tests to setup up exact situations and expectations!